snippets/last.md

```diff
@@ -5,10 +5,10 @@ tags: array,beginner
 
 Returns the last element in an array.
 
-Check if `arr` is truthy and has a `length` property, use `arr.length - 1` to compute the index of the last element of the given array and return it, otherwise return `undefined`.
+Check if `arr` is an instance of an `Array`, use `arr.length - 1` to compute the index of the last element of the given array and return it, otherwise return `undefined`.
 
 ```js
-const last = arr => (arr && arr.length ? arr[arr.length - 1] : undefined);
+const last = arr => (arr instanceof Array ? arr[arr.length - 1] : undefined);
```

Comment:
This needs to be reverted or changed. This is breaking across iframes.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

`When checking for Array instance, Array.isArray is preferred over instanceof because it works through iframes`